bleak/backends/bluezdbus/client.py

```diff
@@ -942,15 +1001,24 @@ async def stop_notify(self, characteristic: BleakGATTCharacteristic) -> None:
 
         assert self._bus is not None
 
-        reply = await self._bus.call(
-            Message(
-                destination=defs.BLUEZ_SERVICE,
-                path=characteristic.obj[0],
-                interface=defs.GATT_CHARACTERISTIC_INTERFACE,
-                member="StopNotify",
+        if "NotifyAcquired" in characteristic.obj[1]:
+            fd = self._notification_fds.pop(characteristic.obj[0])
```

Comment:
```suggestion
            fd = self._notification_fds.pop(characteristic.obj[0], None)
```

I think I was wrong in my previous comment and we do need `None` here to avoid `KeyError` if there is no match.